core/database_arango.py

```diff
@@ -255,16 +256,24 @@ def save(
         if doc_dict.get("id") is not None:
             exclude = ["tags"] + self._exclude_overwrite
             result = self._update(self.model_dump_json(exclude=exclude))
+            event_type = EventType.update
         else:
             exclude = ["tags", "id"] + self._exclude_overwrite
             result = self._insert(self.model_dump_json(exclude=exclude))
+            event_type = EventType.new
             if not result:
                 exclude = exclude_overwrite + self._exclude_overwrite
                 result = self._update(self.model_dump_json(exclude=exclude))
+                event_type = EventType.update
         yeti_object = self.__class__(**result)
         # TODO: Override this if we decide to implement YetiTagModel
         if hasattr(self, "tags"):
             yeti_object.get_tags()
+        try:
+            event = ObjectEvent(type=event_type, yeti_object=yeti_object)
+            producer.publish_event(event)
+        except Exception:
+            logging.exception("Error while publishing event")
```

Comment:
Here and below - it would be good to narrow down the exception a bit; but I understand it's not easily feasible and we really want to be fail-safe. Can we at least add more details to the `logging.exception` call? (i.e. type of error etc.? unless it's already included in the `exception` handler)